source/components/organisms/PinInputModal/PinInputModal.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import Button from "../../atoms/Button";
+import { Heading, Input, Text } from "../../atoms";
+import Wrapper from "../../molecules/Dialog/Wrapper";
+import { Modal } from "../../molecules/Modal";
+import { BackgroundBlur } from "../../atoms/BackgroundBlur";
+import { ButtonContainer, DialogContainer } from "./styled";
+
+export interface PinInputModalProps {
+  visible: boolean;
+  mainApplicantName: string;
+  onClose(): void;
+  onPinEntered(pin: string): void;
+  error?: string;
```

Comment:
**OPTIONAL**

Change the order of properties in the interface like:

```js
export interface PinInputModalProps {
  visible: boolean;
  mainApplicantName: string;
  error?: string;
  onClose(): void;
  onPinEntered(pin: string): void;
}
```

So values comes first and functions lasts